bin/i18n-codeorg/lib/export-without-merge.rb

```diff
@@ -3,28 +3,26 @@
 require 'yaml'
 require 'json'
 
-file_type = ARGV[0]
-new_translation_path = ARGV[1]
-prev_translation_path = ARGV[2]
-
-# is this a new file being translated?
-if File.exist?(new_translation_path) && !File.exist?(prev_translation_path)
-  FileUtils.copy(new_translation_path, prev_translation_path)
-end
+def export_without_merge(file_type, new_translation_path, prev_translation_path)
+  # is this a new file being translated?
+  if File.exist?(new_translation_path) && !File.exist?(prev_translation_path)
+    FileUtils.copy(new_translation_path, prev_translation_path)
+  end
 
-# Translation begins
-if file_type == "yml"
-  new_translation = YAML.load_file(new_translation_path)
+  # Translation begins
+  if file_type == "yml"
```

Comment:
Nit: prefer [single quotes for strings without interpolation](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#ruby-string-quotes).